src/dll/v0/Funcs.cpp

```diff
@@ -1,7 +1,8 @@
-#include "stdafx.hpp"
 #include "Funcs.hpp"
 #include "App.hpp"
 #include "Utils.hpp"
+#include "stdafx.hpp"
```

Comment:
I know this is not added by you, but please remove it. CMake force include this now.

Reply:
nice :)